src/openmrs-react-root-decorator.js

```diff
@@ -1,9 +1,17 @@
 import React from "react";
+import { ModuleNameContext } from "@openmrs/esm-module-config";
 
 export default function decorateOptions(opts) {
-  if (typeof opts !== "object" || typeof opts.featureName !== "string") {
+  if (
+    typeof opts !== "object" ||
+    typeof opts.featureName !== "string" ||
+    typeof opts.moduleName !== "string"
+  ) {
     throw new Error(
-      "openmrs-react-root-decorator should be called with an opts object that has a featureName string. e.g. @ErrorBoundary({featureName: 'life'})"
+      "openmrs-react-root-decorator should be called with an opts object that has " +
```

Comment:
Are you not using a template literal for this string because you don't want the new lines?